Monika After Story/game/script-topics.rpy

```diff
@@ -1265,6 +1265,10 @@ label monika_portraitof:
     m 2lksdla "Actually, I don't think I should be talking about this."
     m 2hksdlb "Ahaha, sorry!"
     m 1rksdla "Just forget I said anything."
+
+    # Remove previous 7 lines from history.
+    $ del _history_list[-7:]
```

Comment:
We're usually only removing the lines we don't want the user to see, not the whole conversation.
```renpy
    m 4eua "Hey, you know that book you were reading with Yuri?"
    m "Portrait of...whatever it was called..."
    m 4hub "It's funny, because I'm pretty sure that book--"
    m 1wuw "Ah..."
    $ del _history_list[-4:]
```
First 3 lines she talks about the book. Then she recalls she shouldn't talk about it. Then decides to delete the lines above. And then explains why.